### ISSUE TITLE
console spam is insane

### ISSUE THREAD

**Issue by CrazyMonkeyMcgee** [TheXFactor117/Levels#131]
Every time I go to check on server logs >90% is just levels changing values. Is there really a reason that levels needs to print every time something changes?

**Comment by TheXFactor117** [comment]
Nope, they are for my purposes. I'll clean it up once I verify everything is working properly.

**Comment by CrazyMonkeyMcgee** [comment]
Thanks for the quick reply. It just makes it hard to diagnose conflicts and help players when my logs are filled with levels levels levels.

**Comment by curscascis** [comment]
Hey xFactor, would you be able to point where in the code the logging is being triggered. I'm on version 1.10.2 r1.2.8 but I'd like to remove the logging and rebuild it so that I can have clean logs on my server. I was looking but I'm not too certain where exactly it is. It seemed like it was in the level up but I'm unsure.

**Comment by mkrtes** [comment]
I have it too terribly annoying

**Comment by TheXFactor117** [comment]
Anywhere that says Levels.LOGGER is an output. That's my fault for not removing them, sorry about that.
